src/components/CompletionForcast.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRef } from "react";
+import { Overlay, Tooltip } from "react-bootstrap";
+
+
+export function CompletionForcast({CompletionForcastEval,themeColors, toggleHelpTips}){
```

Comment:
what the eval stands for? 
you don't really need to repeat the CompletionForcast anytime

Reply:
CompletionForcastEval is a function that return a value that represent the current rate - its the completion forecast rate basically/.